rpcs3/Emu/Memory/vm.cpp

```diff
@@ -744,7 +744,7 @@ namespace vm
 			utils::memory_commit(g_exec_addr + addr * 2, size * 2);
 		}
 
-		if (g_cfg.core.ppu_debug)
+		if (g_cfg.core.ppu_debug && addr < 0x2000'0000)
```

Comment:
Perhaps can be moved to exec allocation range function.

Reply:
Moved.